corehq/apps/hqwebapp/static/hqwebapp/js/knockout_bindings.ko.js

```diff
@@ -664,9 +664,17 @@ hqDefine("hqwebapp/js/knockout_bindings.ko", [
     };
 
     ko.bindingHandlers.sortableList = {
-        init: function (element, valueAccessor) {
-            var list = valueAccessor();
+        // Defines a knockout binding for the jquery UI sortable interaction which allows
+        // reordering elements with a drag and drop interface
+        // The element to be used to drag and drop must have a grip class defined
+        // Optionally sortableListItems can be defined as a selector that describes what items are sortable
```

Comment:
I think calling this something like `sortableItemsSelector` would be a bit less confusing. I was expecting `sortableListItems` to actually be... a list of sortable items.

Reply:
Originally did this because jquery ui select uses 'items' I changed this to `sortableListSelector` instead of `sortableItemsSelector` as I think `sortableList` should be the namespace for the binding. https://github.com/dimagi/commcare-hq/pull/23689/commits/58d0786ae5bfb9b7ff04d00bf8a84c90839d8bce